docs/remote/containers.md

```diff
@@ -98,7 +98,11 @@ This quick start covers how to set up a dev container for an existing project to
 
     ![Select a node dev container definition](images/containers/select-dev-container-def.png)
 
-    The list will be automatically sorted based on the contents of the folder you open. Note the dev container definitions displayed come from the [vscode-dev-containers repository](https://aka.ms/vscode-dev-containers). You can browse the `containers` folder of that repository to see the contents of each definition.
+    The list will be automatically sorted based on the contents of the folder you open.
+
+    You will also be able to customize your dev container with additional features, which [you can read more about below](#Dev-container-features).
```

Comment:
We should probably say you "may" be able to pick from features since not all of the definitions have them.